Entity/Dynamic.php

```diff
@@ -170,6 +176,10 @@ public function getData()
      */
     public function __set($name, $value)
     {
+        if ($value instanceof \DateTime) {
+            $value = $value->format('Y-m-d H:i:s');
+        }
+
         if (property_exists($this, $name)) {
             if (is_array($value)) {
                 $value = json_encode($value);
```

Comment:
in `__get` #L203 the json_decode($value, true) is missing when type `multipleChoice`.